proposals/incubation/credebl.md

```diff
@@ -0,0 +1,127 @@
+
+CREDEBL is a production-ready project with a robust 1.0 release with multiple production deployments at national scale.
+
+Our team has reviewed the Project Incubation Exit Criteria and believes we fulfill all the requirements
+for "Graduated" status, with the exception of being on-boarded onto Linux Foundation’s tools and build processes.
+
+We are prepared to complete these steps promptly following an approval vote by the Technical Advisory Council (TAC).
+
+Once these final steps are completed, we believe the project will be ready for "Graduated" status and will apply for
+this designation immediately.
+
+We anticipate that this designation process will proceed swiftly if the TAC concurs that all criteria have been met.
+
+
+# Solution
+The CREDEBL Codebase is currently hosted as a separate Github Org at https://github.com/credebl
+
+### CREDEBL PLATFORM
+Platform provides a set of scalable services to manage decentralized identity and verifiable credentials (VC) system. It enables developers to build Self-Sovereign Identity (SSI) solutions that provide enhanced privacy and security. It is designed for scalability to support population-scale SSI implementations. 
+
+It provides tools for digital credential management and verification, allowing organizations to easily issue and manage credentials with ease. It is built on a micro-services architecture to support large-scale enterprise applications.
+
+Platform supports multiple ledger options (like Indy and Polygon) and also ledger-less credential issuance using methods like did:web, did:key and did:peer.
+
+### CREDEBL STUDIO
+Studio is web front-end for the entire Platform, built using Astro, React, Flowbite, and Tailwind CSS, allowing administrators & users to easily access & manage all the Platform features using a simple web interface.
+
+### ARIES AGENT
+In the context of decentralized identity systems, an agent generally refers to a software component or service that facilitates peer-to-peer interactions and data exchange operations between various entities in the system.
+
+Agents leverage the Aries protocol implementations built into Credo & Aca-Py.
+
+There are two types of agents that organizations can use, each serving different needs based on the deployment model and use case: Shared Agents and Dedicated Agents.
+
+A shared agent refers to an agent that is used by multiple entities or tenants, rather than being dedicated to a single entity. If the tenancy is set to true, it typically means that the system or agent supports multi-tenancy, or if the tenancy is set to false, it means that the agent does not support the multi-tenancy.
+
+### Mobile SDK
+Built on the strong foundation of Credo, the Mobile Wallet SDK is a React-Native SDK helps mobile developers to easily build SSI features with into existing mobile apps or a new one, enables interoperability and seamless connectivity for secure data exchange.
```

Comment:
Similar question here as above. Can you be more specific about the usage of Credo? Is the Mobile Wallet SDK a Credo wrapper or something else?

Reply:
Yes, the mobile SDK is a wrapper built on top of Credo - having alignment to CREDEBL features.